test/sanity/ignore.txt

```diff
@@ -6914,6 +6914,7 @@ lib/ansible/plugins/action/netconf.py action-plugin-docs # base class for deprec
 lib/ansible/plugins/action/network.py action-plugin-docs # base class for network action plugins
 lib/ansible/plugins/action/normal.py action-plugin-docs # default action plugin for modules without a dedicated action plugin
 lib/ansible/plugins/action/nxos.py action-plugin-docs # base class for deprecated network platform modules using `connection: local`
+lib/ansible/plugins/action/qnos.py action-plugin-docs # base class for deprecated network platform modules using `connection: local`
```

Comment:
Why is new code being added that depends on deprecated functionality?

Reply:
As the local connection (provider option) is deprecated and is not supported by qnos (followed ganeshrn’s suggestion), no qnos.py exists at plugins/doc_fragments.
Adding this line to skip the checking ‘ansible-test sanity --test action-plugin-docs.’